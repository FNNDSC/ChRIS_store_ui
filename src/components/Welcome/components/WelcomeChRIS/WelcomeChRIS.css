.welcome-chris {
  /* color */
  background: #39a5dc;
  background: var(--pf-blue-300);

  /* display */
  min-height: 400px;
  padding: 2em;

  /* position */
  position: relative;
}

.welcome-chris .row > div {
  /* display */
  padding: 1em;

  /* flex */
  flex: 1;
}

.welcome-chris-text {
  /* color */
  color: #fafafa;
  color: var(--pf-black-100);
}

.welcome-chris-text-container > div {
  font-weight: 400;
  font-size: 3em;
}

.welcome-chris-text-container > p { font-size: 1.3em; }

.welcome-chris-video-column {
  /* display */
  display: flex;

  /* flex */
  flex-direction: column;
  justify-content: center;
}

.welcome-chris-video {
  /* display */
  width: 100%;
  height: 200px;
}

.welcome-chris-video-desc {
  /* color */
  color: #fafafa;
  color: var(--pf-black-100);

  /* display */
  padding: 1em 1em 0 0;
}

.welcome-chris-btn {
  /* display */
  display: block !important;
  width: 100%;
}

.welcome-chris-banner { display: none; }

/* ============================== */
/* ---------- RESPONSIVE -------- */
/* ============================== */

@media (min-width: 420px) {
  .welcome-chris-btn-row > div { display: inline-block; }

  .welcome-chris-btn { width: auto; }
}

@media (min-width: 480px)  {
  .welcome-chris-video { height: 50vw; }
}

@media (min-width: 768px)  {
  .welcome-chris-video {
    /* display */
    width: 45vw;
    height: 25.3vw;
  }

  .welcome-chris-more-btn {
    /* display */
    width: 208.5px;

    /* other */
    float: right;
  }

  .welcome-chris > .row:first-child { flex-direction: row-reverse; }
}

@media (min-width: 1108px) {
  .welcome-chris-video {
    /* display */
    width: 500px;
    height: 281.25px;
  }

  .welcome-chris-text-container {
    /* display */
    padding: 0 2em 1.1em 2em;
    width: 33vw;
  }

  .welcome-chris-banner {
    /* display */
    display: block;

    /* position */
    position: absolute;
    top: 0;
    right: 5em;
  }
}

@media (min-width: 1267px) {
  .welcome-chris-text-container { width: auto; }
}
