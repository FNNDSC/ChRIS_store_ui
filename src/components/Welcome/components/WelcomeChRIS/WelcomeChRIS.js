import React from 'react';
import { Button } from 'patternfly-react';
import './WelcomeChRIS.css';
import chrisBanner from '../../../../assets/img/chris_logo-ribbon.png';

const WelcomeChRIS = () => (
  <div className="welcome-chris">
    <div className="row">
      <div className="welcome-chris-text">
        <div className="welcome-chris-text-container">
          <div>
            What is ChRIS?
          </div>
          <p>
            ChRIS is an open source framework that utilizes cloud technologies to
            democratize medical analytics application development and enables
            healthcare organizations to keep owning their data while benefiting from
            public cloud processing capabilities.
          </p>
          <p>
            ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other
            technologies to standardize healthcare application development.
          </p>
        </div>
      </div>
      <div className="welcome-chris-video-column">
        <div>
          <iframe
            title="ChRIS video"
            className="welcome-chris-video"
            src="https://www.youtube-nocookie.com/embed/dyFQD87jU68"
            frameBorder="0"
            allow="autoplay; encrypted-media"
            allowFullScreen
          />
          <p className="welcome-chris-video-desc">
            {'"'}Creating ChRIS{'"'} is an upcoming video series by Red Hat films that
            focuses on the ChRIS project - its purpose, development, milestones.
          </p>
        </div>
      </div>
    </div>
    <div className="row welcome-chris-btn-row">
      <div>
        <Button
          bsStyle="default"
          bsSize="large"
          className="welcome-chris-btn welcome-chris-more-btn"
        >
          Learn more
        </Button>
      </div>
      <div>
        <Button
          bsStyle="default"
          bsSize="large"
          className="welcome-chris-btn"
        >
          Don{"'"}t have access to ChRIS?
        </Button>
      </div>
    </div>
    <span>
      <img src={chrisBanner} alt="" className="welcome-chris-banner" />
    </span>
  </div>
);

export default WelcomeChRIS;
