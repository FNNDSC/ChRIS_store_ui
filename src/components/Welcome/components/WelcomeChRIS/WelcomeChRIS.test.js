import React from 'react';
import { shallow } from 'enzyme';
import WelcomeChRIS from './WelcomeChRIS';

describe('WelcomeChRIS', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<WelcomeChRIS />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render welcome-chris div', () => {
    expect(wrapper.find('div.welcome-chris')).toHaveLength(1);
  });

  it('should render 2 row divs', () => {
    expect(wrapper.find('div.row')).toHaveLength(2);
  });

  it('should render welcome-chris-text div', () => {
    expect(wrapper.find('div.welcome-chris-text')).toHaveLength(1);
  });

  it('should render welcome-chris-text-container', () => {
    expect(wrapper.find('div.welcome-chris-text-container')).toHaveLength(1);
  });

  it('should render welcome-chris-video-column', () => {
    expect(wrapper.find('div.welcome-chris-video-column')).toHaveLength(1);
  });

  it('should render welcome-chris-video iframe', () => {
    expect(wrapper.find('iframe.welcome-chris-video')).toHaveLength(1);
  });

  it('should render welcome-chris-video desc p element', () => {
    expect(wrapper.find('p.welcome-chris-video-desc')).toHaveLength(1);
  });

  it('should render welcome-chris-btn-row div', () => {
    expect(wrapper.find('div.welcome-chris-btn-row')).toHaveLength(1);
  });

  it('should render 2 Button components in welcome-chris-btn-row', () => {
    expect(wrapper
      .find('div.welcome-chris-btn-row')
      .find('Button').length)
      .toEqual(2);
  });

  it('should render span inside welcome-chris div', () => {
    expect(wrapper
      .find('div.welcome-chris')
      .find('span')).toHaveLength(1);
  });

  it('should render welcome-chris-banner img', () => {
    expect(wrapper.find('img.welcome-chris-banner')).toHaveLength(1);
  });
});
