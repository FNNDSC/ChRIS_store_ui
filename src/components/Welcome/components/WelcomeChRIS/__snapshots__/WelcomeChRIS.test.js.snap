// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeChRIS should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WelcomeChRIS />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="row"
>
          <div
                    className="welcome-chris-text"
          >
                    <div
                              className="welcome-chris-text-container"
                    >
                              <div>
                                        What is ChRIS?
                              </div>
                              <p>
                                        ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.
                              </p>
                              <p>
                                        ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.
                              </p>
                    </div>
          </div>
          <div
                    className="welcome-chris-video-column"
          >
                    <div>
                              <iframe
                                        allow="autoplay; encrypted-media"
                                        allowFullScreen={true}
                                        className="welcome-chris-video"
                                        frameBorder="0"
                                        src="https://www.youtube-nocookie.com/embed/dyFQD87jU68"
                                        title="ChRIS video"
                              />
                              <p
                                        className="welcome-chris-video-desc"
                              >
                                        "
                                        Creating ChRIS
                                        "
                                         is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.
                              </p>
                    </div>
          </div>
</div>,
        <div
          className="row welcome-chris-btn-row"
>
          <div>
                    <Button
                              active={false}
                              block={false}
                              bsClass="btn"
                              bsSize="large"
                              bsStyle="default"
                              className="welcome-chris-btn welcome-chris-more-btn"
                              disabled={false}
                    >
                              Learn more
                    </Button>
          </div>
          <div>
                    <Button
                              active={false}
                              block={false}
                              bsClass="btn"
                              bsSize="large"
                              bsStyle="default"
                              className="welcome-chris-btn"
                              disabled={false}
                    >
                              Don
                              '
                              t have access to ChRIS?
                    </Button>
          </div>
</div>,
        <span>
          <img
                    alt=""
                    className="welcome-chris-banner"
                    src="chris_logo-ribbon.png"
          />
</span>,
      ],
      "className": "welcome-chris",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="welcome-chris-text"
>
              <div
                            className="welcome-chris-text-container"
              >
                            <div>
                                          What is ChRIS?
                            </div>
                            <p>
                                          ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.
                            </p>
                            <p>
                                          ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.
                            </p>
              </div>
</div>,
            <div
              className="welcome-chris-video-column"
>
              <div>
                            <iframe
                                          allow="autoplay; encrypted-media"
                                          allowFullScreen={true}
                                          className="welcome-chris-video"
                                          frameBorder="0"
                                          src="https://www.youtube-nocookie.com/embed/dyFQD87jU68"
                                          title="ChRIS video"
                            />
                            <p
                                          className="welcome-chris-video-desc"
                            >
                                          "
                                          Creating ChRIS
                                          "
                                           is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.
                            </p>
              </div>
</div>,
          ],
          "className": "row",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="welcome-chris-text-container"
>
                <div>
                                What is ChRIS?
                </div>
                <p>
                                ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.
                </p>
                <p>
                                ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.
                </p>
</div>,
              "className": "welcome-chris-text",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div>
                    What is ChRIS?
</div>,
                  <p>
                    ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.
</p>,
                  <p>
                    ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.
</p>,
                ],
                "className": "welcome-chris-text-container",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "What is ChRIS?",
                  },
                  "ref": null,
                  "rendered": "What is ChRIS?",
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.",
                  },
                  "ref": null,
                  "rendered": "ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.",
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.",
                  },
                  "ref": null,
                  "rendered": "ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div>
                <iframe
                                allow="autoplay; encrypted-media"
                                allowFullScreen={true}
                                className="welcome-chris-video"
                                frameBorder="0"
                                src="https://www.youtube-nocookie.com/embed/dyFQD87jU68"
                                title="ChRIS video"
                />
                <p
                                className="welcome-chris-video-desc"
                >
                                "
                                Creating ChRIS
                                "
                                 is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.
                </p>
</div>,
              "className": "welcome-chris-video-column",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <iframe
                    allow="autoplay; encrypted-media"
                    allowFullScreen={true}
                    className="welcome-chris-video"
                    frameBorder="0"
                    src="https://www.youtube-nocookie.com/embed/dyFQD87jU68"
                    title="ChRIS video"
/>,
                  <p
                    className="welcome-chris-video-desc"
>
                    "
                    Creating ChRIS
                    "
                     is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.
</p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "allow": "autoplay; encrypted-media",
                    "allowFullScreen": true,
                    "className": "welcome-chris-video",
                    "frameBorder": "0",
                    "src": "https://www.youtube-nocookie.com/embed/dyFQD87jU68",
                    "title": "ChRIS video",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "iframe",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      "\\"",
                      "Creating ChRIS",
                      "\\"",
                      " is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.",
                    ],
                    "className": "welcome-chris-video-desc",
                  },
                  "ref": null,
                  "rendered": Array [
                    "\\"",
                    "Creating ChRIS",
                    "\\"",
                    " is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.",
                  ],
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              <Button
                            active={false}
                            block={false}
                            bsClass="btn"
                            bsSize="large"
                            bsStyle="default"
                            className="welcome-chris-btn welcome-chris-more-btn"
                            disabled={false}
              >
                            Learn more
              </Button>
</div>,
            <div>
              <Button
                            active={false}
                            block={false}
                            bsClass="btn"
                            bsSize="large"
                            bsStyle="default"
                            className="welcome-chris-btn"
                            disabled={false}
              >
                            Don
                            '
                            t have access to ChRIS?
              </Button>
</div>,
          ],
          "className": "row welcome-chris-btn-row",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Button
                active={false}
                block={false}
                bsClass="btn"
                bsSize="large"
                bsStyle="default"
                className="welcome-chris-btn welcome-chris-more-btn"
                disabled={false}
>
                Learn more
</Button>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "active": false,
                "block": false,
                "bsClass": "btn",
                "bsSize": "large",
                "bsStyle": "default",
                "children": "Learn more",
                "className": "welcome-chris-btn welcome-chris-more-btn",
                "disabled": false,
              },
              "ref": null,
              "rendered": "Learn more",
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Button
                active={false}
                block={false}
                bsClass="btn"
                bsSize="large"
                bsStyle="default"
                className="welcome-chris-btn"
                disabled={false}
>
                Don
                '
                t have access to ChRIS?
</Button>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "active": false,
                "block": false,
                "bsClass": "btn",
                "bsSize": "large",
                "bsStyle": "default",
                "children": Array [
                  "Don",
                  "'",
                  "t have access to ChRIS?",
                ],
                "className": "welcome-chris-btn",
                "disabled": false,
              },
              "ref": null,
              "rendered": Array [
                "Don",
                "'",
                "t have access to ChRIS?",
              ],
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt=""
            className="welcome-chris-banner"
            src="chris_logo-ribbon.png"
/>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "",
            "className": "welcome-chris-banner",
            "src": "chris_logo-ribbon.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="row"
>
            <div
                        className="welcome-chris-text"
            >
                        <div
                                    className="welcome-chris-text-container"
                        >
                                    <div>
                                                What is ChRIS?
                                    </div>
                                    <p>
                                                ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.
                                    </p>
                                    <p>
                                                ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.
                                    </p>
                        </div>
            </div>
            <div
                        className="welcome-chris-video-column"
            >
                        <div>
                                    <iframe
                                                allow="autoplay; encrypted-media"
                                                allowFullScreen={true}
                                                className="welcome-chris-video"
                                                frameBorder="0"
                                                src="https://www.youtube-nocookie.com/embed/dyFQD87jU68"
                                                title="ChRIS video"
                                    />
                                    <p
                                                className="welcome-chris-video-desc"
                                    >
                                                "
                                                Creating ChRIS
                                                "
                                                 is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.
                                    </p>
                        </div>
            </div>
</div>,
          <div
            className="row welcome-chris-btn-row"
>
            <div>
                        <Button
                                    active={false}
                                    block={false}
                                    bsClass="btn"
                                    bsSize="large"
                                    bsStyle="default"
                                    className="welcome-chris-btn welcome-chris-more-btn"
                                    disabled={false}
                        >
                                    Learn more
                        </Button>
            </div>
            <div>
                        <Button
                                    active={false}
                                    block={false}
                                    bsClass="btn"
                                    bsSize="large"
                                    bsStyle="default"
                                    className="welcome-chris-btn"
                                    disabled={false}
                        >
                                    Don
                                    '
                                    t have access to ChRIS?
                        </Button>
            </div>
</div>,
          <span>
            <img
                        alt=""
                        className="welcome-chris-banner"
                        src="chris_logo-ribbon.png"
            />
</span>,
        ],
        "className": "welcome-chris",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="welcome-chris-text"
>
                <div
                                className="welcome-chris-text-container"
                >
                                <div>
                                                What is ChRIS?
                                </div>
                                <p>
                                                ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.
                                </p>
                                <p>
                                                ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.
                                </p>
                </div>
</div>,
              <div
                className="welcome-chris-video-column"
>
                <div>
                                <iframe
                                                allow="autoplay; encrypted-media"
                                                allowFullScreen={true}
                                                className="welcome-chris-video"
                                                frameBorder="0"
                                                src="https://www.youtube-nocookie.com/embed/dyFQD87jU68"
                                                title="ChRIS video"
                                />
                                <p
                                                className="welcome-chris-video-desc"
                                >
                                                "
                                                Creating ChRIS
                                                "
                                                 is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.
                                </p>
                </div>
</div>,
            ],
            "className": "row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="welcome-chris-text-container"
>
                  <div>
                                    What is ChRIS?
                  </div>
                  <p>
                                    ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.
                  </p>
                  <p>
                                    ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.
                  </p>
</div>,
                "className": "welcome-chris-text",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <div>
                      What is ChRIS?
</div>,
                    <p>
                      ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.
</p>,
                    <p>
                      ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.
</p>,
                  ],
                  "className": "welcome-chris-text-container",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "What is ChRIS?",
                    },
                    "ref": null,
                    "rendered": "What is ChRIS?",
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.",
                    },
                    "ref": null,
                    "rendered": "ChRIS is an open source framework that utilizes cloud technologies to democratize medical analytics application development and enables healthcare organizations to keep owning their data while benefiting from public cloud processing capabilities.",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.",
                    },
                    "ref": null,
                    "rendered": "ChRIS uses Docker/containers, Kubernetes/OpenShift, and a set of other technologies to standardize healthcare application development.",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div>
                  <iframe
                                    allow="autoplay; encrypted-media"
                                    allowFullScreen={true}
                                    className="welcome-chris-video"
                                    frameBorder="0"
                                    src="https://www.youtube-nocookie.com/embed/dyFQD87jU68"
                                    title="ChRIS video"
                  />
                  <p
                                    className="welcome-chris-video-desc"
                  >
                                    "
                                    Creating ChRIS
                                    "
                                     is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.
                  </p>
</div>,
                "className": "welcome-chris-video-column",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <iframe
                      allow="autoplay; encrypted-media"
                      allowFullScreen={true}
                      className="welcome-chris-video"
                      frameBorder="0"
                      src="https://www.youtube-nocookie.com/embed/dyFQD87jU68"
                      title="ChRIS video"
/>,
                    <p
                      className="welcome-chris-video-desc"
>
                      "
                      Creating ChRIS
                      "
                       is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.
</p>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "allow": "autoplay; encrypted-media",
                      "allowFullScreen": true,
                      "className": "welcome-chris-video",
                      "frameBorder": "0",
                      "src": "https://www.youtube-nocookie.com/embed/dyFQD87jU68",
                      "title": "ChRIS video",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "iframe",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "\\"",
                        "Creating ChRIS",
                        "\\"",
                        " is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.",
                      ],
                      "className": "welcome-chris-video-desc",
                    },
                    "ref": null,
                    "rendered": Array [
                      "\\"",
                      "Creating ChRIS",
                      "\\"",
                      " is an upcoming video series by Red Hat films that focuses on the ChRIS project - its purpose, development, milestones.",
                    ],
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                <Button
                                active={false}
                                block={false}
                                bsClass="btn"
                                bsSize="large"
                                bsStyle="default"
                                className="welcome-chris-btn welcome-chris-more-btn"
                                disabled={false}
                >
                                Learn more
                </Button>
</div>,
              <div>
                <Button
                                active={false}
                                block={false}
                                bsClass="btn"
                                bsSize="large"
                                bsStyle="default"
                                className="welcome-chris-btn"
                                disabled={false}
                >
                                Don
                                '
                                t have access to ChRIS?
                </Button>
</div>,
            ],
            "className": "row welcome-chris-btn-row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Button
                  active={false}
                  block={false}
                  bsClass="btn"
                  bsSize="large"
                  bsStyle="default"
                  className="welcome-chris-btn welcome-chris-more-btn"
                  disabled={false}
>
                  Learn more
</Button>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "active": false,
                  "block": false,
                  "bsClass": "btn",
                  "bsSize": "large",
                  "bsStyle": "default",
                  "children": "Learn more",
                  "className": "welcome-chris-btn welcome-chris-more-btn",
                  "disabled": false,
                },
                "ref": null,
                "rendered": "Learn more",
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Button
                  active={false}
                  block={false}
                  bsClass="btn"
                  bsSize="large"
                  bsStyle="default"
                  className="welcome-chris-btn"
                  disabled={false}
>
                  Don
                  '
                  t have access to ChRIS?
</Button>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "active": false,
                  "block": false,
                  "bsClass": "btn",
                  "bsSize": "large",
                  "bsStyle": "default",
                  "children": Array [
                    "Don",
                    "'",
                    "t have access to ChRIS?",
                  ],
                  "className": "welcome-chris-btn",
                  "disabled": false,
                },
                "ref": null,
                "rendered": Array [
                  "Don",
                  "'",
                  "t have access to ChRIS?",
                ],
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt=""
              className="welcome-chris-banner"
              src="chris_logo-ribbon.png"
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "",
              "className": "welcome-chris-banner",
              "src": "chris_logo-ribbon.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
